webpack.config.js

```diff
@@ -39,13 +36,16 @@ function createConfig(isDebug) {
     config.debug = true;
   } else {
     config.output.filename = 'enigma.min.js';
-    config.plugins.push(new Webpack.optimize.UglifyJsPlugin());
+    config.plugins.push(new Webpack.optimize.UglifyJsPlugin({
```

Comment:
Looking at the CircleCI artifacts here: https://circleci.com/gh/qlik-oss/enigma.js/8#artifacts/containers/0

Compared to the latest master build: https://circleci.com/gh/qlik-oss/enigma.js/6#artifacts/containers/0